packages/connect-conformance/src/cloudflare/compression.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021-2024 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Code, ConnectError } from "@connectrpc/connect";
+import type { Compression } from "@connectrpc/connect/protocol";
+
+export const compressionGzip = createCompression("gzip");
+export const compressionDeflate = {
```

Comment:
Just for my understanding: I assume we need those for (de-)compressing individual envelope payloads, and rely on automatic full-body decompression by the runtime's fetch implementation for Connect unary?

Reply:
Yes, that's correct